api/turing/service/router_deployment_service.go

```diff
@@ -771,13 +798,17 @@ func (ds *deploymentService) createPodDisruptionBudgets(
 	}
 
 	// Router's PDB
-	routerPdb := ds.svcBuilder.NewPodDisruptionBudget(
-		routerVersion,
-		project,
-		servicebuilder.ComponentTypes.Router,
-		ds.pdbConfig,
-	)
-	pdbs = append(pdbs, routerPdb)
+	if routerVersion.ResourceRequest != nil &&
```

Comment:
Is it possible that the resource request be `nil`?

Reply:
It's not 😅 and I don't think we've cared about this in many parts of the code. But simply because we are accessing a pointer, I thought it would be better to include the check.